src/components/Footer.css

```
@@ -0,0 +1,71 @@
+@import url('../styles/colors.css');
+
+footer {
+  background-color: var(--black-color);
+}
+
+.footer_wrap {
+  padding: 32px 104px 108px;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  align-items: center;
+  color: #676767;
+}
+
+.privacy_fnq_wrap {
+  justify-self: center;
+}
+
+.sns_fill_wrap {
+  justify-self: end;
+}
+
+.footer_wrap a {
+  color: #cfcfcf;
+}
+
+.privacy_fnq_wrap {
+  display: flex;
+  align-items: center;
+  gap: 30px;
+}
+
+.sns_fill_wrap {
+  display: flex;
+  gap: 12px;
+}
+
+@media screen and (max-width: 1124px) {
+  .footer_wrap {
+    padding: 32px 104px 108px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 1fr;
+    justify-items: center;
+    gap: 20px;
+    color: #676767;
+  }
+  .sns_fill_wrap {
```

여기도 다른 곳들과 동일하게 한 칸 떼주세용 
```suggestion
  }
  
  .sns_fill_wrap {
```